src/components/Search/SearchComponent.tsx

```diff
@@ -171,7 +171,10 @@ function SearchComponent({
       </Grid>
       {/***** Selected Revisions Section *****/}
       {displayedRevisions && (
-        <Grid className='d-flex'>
+        <Grid
+          className='d-flex'
+          data-testid={`${isBaseComp ? 'base' : 'new'}-selected-revision`}
```

Comment:
nit: reuse `searchType` directly instead of using the ternary

Reply:
True, forgot about that